compiler/p/codegen/OMRMemoryReference.cpp

```diff
@@ -1699,11 +1711,26 @@ void OMR::Power::MemoryReference::accessStaticItem(TR::Node *node, TR::SymbolRef
 
          if (comp->getOption(TR_AOT))
```

Comment:
i know it's not your fault, but this isn't really the right condition to use :(

Reply:
Heh, if that makes you sad, you should probably not grep for all the other instances of that query ;P. At any rate I'll fix it to use the canonical `compileRelocatableCode()` query.